internal/tasks/asset_resize.go

```diff
@@ -104,22 +104,12 @@ func (c Context) ExecuteNextResize() (targetAssetType db.AssetType, returnedErro
 	outcome, err := manager.SetAssetSize(res, asset.UUID, asset.Size, op.NewSize)
 	errorMessage := ""
 	if err != nil {
-		e := setAssetSizeError{
-			scopeUUID: res.ScopeUUID,
-			assetType: string(res.AssetType),
-			assetUUID: asset.UUID,
-			newSize:   op.NewSize,
-			inner:     err,
-		}
-		logg.Error(e.Error())
+		logg.Error("cannot resize %s %s to size %d: %s", string(res.AssetType), asset.UUID, op.NewSize, err.Error())
 
-		//do not log "failed" operations to Sentry: fixing them is not within the
-		//responsibility of the Castellum operator
-		if c.SentryHub != nil && outcome == db.OperationOutcomeErrored {
-			captureSentryException(c.SentryHub, e)
+		if outcome == db.OperationOutcomeErrored {
+			//We are only interested in tracking the status errored in the DB.
```

Comment:
No, we also need error messages in the DB for the "failed" operations. Otherwise users won't be able to see error messages on the `failed-operations` API endpoint.

Reply:
Concerning commit has been reverted.